app/controllers/admin/content_controller.rb

```diff
@@ -37,6 +37,24 @@ def edit
     new_or_edit
   end
 
+  def reverse
+    @article = Article.find(params[:id])
+    if @article != nil
+      text = @article.body
+      split_text = text.split
+      split_text.reverse!
+      reverse_string = ""
+      split_text.each do |word|
```

Comment:
I would like to challenge you to use an Enumerable method to do this for you rather than the each :-D